en/docs/testing/unit-testing-in-build-pipeline.md

```diff
@@ -0,0 +1,28 @@
+# Unit Testing in Choreo Build Pipeline
+
+Choreo supports unit testing as part of the build pipeline, enabling users to validate their applications before deployment. This feature ensures that code changes are tested early, reducing the risk of defects in production.
+
+!!! note
+        Currently, Choreo only supports unit testing for the `WSO2 MI` buildpack.
+
+## How It Works
+
+### Develop tests
```

Comment:
"Write" and "create" are terms commonly used in reference to unit tests. Shall we use one of those terms instead of “Develop”?